tensorflow/compiler/xla/client/client_library.h

```diff
@@ -73,8 +82,11 @@ class ClientLibrary {
   //
   //   platform : The platform the underlying XLA service should target. If
   //     null then default platform is used.
+  //   device_set: Set of device IDs for which the stream executor will be
+  //   created for, for the given platform.
```

Comment:
"for which X will be created for" -> "for which X will be created" or "which X will be created for"

Reply:
Done.